Usings paths instead of URLs on docs about redirects (#552)
vtex-apps/io-documentation: docs/en/Recipes/store-management/managing-url-redirects.md

![csv-file-url-redirect](https://user-images.githubusercontent.com/52087100/78804703-dabb8880-7996-11ea-9fa6-2acd8766f67d.png)

| Property name | Description | Example |
|--|--|--|

> | `from` | Original path, the path you want to create a redirect *from* | `/blouse/p` |
> | `to` | New path or URL the one users will be redirected *to* | `/blouse/p?skuId=2000549` |
> | `type` | The type of the redirect (`TEMPORARY` or `PERMANENT`) | `TEMPORARY` |
> | `endDate` | Date, on the format `mm/dd/yyyy` at which the redirect will expire. Notice that if the redirect is permanent, no value for `endDate` is required. | `5/20/2020` |

Reviewer: ```suggestion
| `from` | Original path i.e. the path you want to create a redirect *from* | `/blouse/p` |
| `to` | New path or URL where users will be redirected *to*  | `/blouse/p?skuId=2000549` |
| `type` | The type of the redirect (`TEMPORARY` or `PERMANENT`) | `TEMPORARY` |
| `endDate` | Date (following the format `mm/dd/yyyy`) on which the redirect will expire. Notice that if the redirect is permanent, no value for `endDate` is required. | `5/20/2020` |
```

> > Only the `to` columns accepts domains, like *myotherstore.com* ou *mystore.myvtex.com*. Do not use them on the `from` column! 

Reviewer: ```suggestion
<div class="alert alert-info">
<strong>Only</strong> the <code>to</code> column accepts domains, such as <code>myotherstore.com</code> or <code>mystore.myvtex.com</code>. You should not use them in the <code>from</code> column. 
</div>
```

</div>

3. Fill in the cells under each column with the values desired for your store. Remember that each redirect's data must be entered on different lines, as shown in the example below: 

![urls-redirect-csv-file](https://user-images.githubusercontent.com/18706156/78902791-a3111700-7a50-11ea-8089-c4fe094a51fb.png)

4. Save and name the document as desired;
5. Using your terminal and the [VTEX IO Toolbelt](https://vtex.io/docs/recipes/development/vtex-io-cli-installment-and-command-reference), log into your VTEX account;
6. Once logged in, run the `vtex redirects import {fileName}.csv`, replacing `{fileName}` with the name of the file that you've just saved.

After the sixth step, Toolbelt will receive the file, validate it, and save it. After that, the redirects will take effect.

### Removing redirects

1. Using your terminal and [VTEX IO Toolbelt](https://vtex.io/docs/recipes/development/vtex-io-cli-installment-and-command-reference), log into your VTEX account;
2. Once logged in, run the `vtex redirects delete {path}`, replacing`{path}` with the URL path that will have its redirect removed.

<div class="alert alert-warning">

> The path that is sent when running <code>vtex redirects delete {path}</code> will have its redirects removed, so it must be one of paths from the `from` column of the <code>.csv</code> type file.

Reviewer: ```suggestion
The path that is sent when running <code>vtex redirects delete {path}</code> will have its redirect removed, so it must be one of paths from the `from` column of the <code>.csv</code> type file.
```
